### DISCUSSION TITLE
Telluric lines

### DISCUSSION THREAD

**Discussion by Ramiapp** [PyWiFeS/pywifes#70] [Q&A]
If the reduction runs and there are many telluric lines that survive and show up in the final cube, what steps are responsible for removing them when the observation mode is “ClassicalEqual”?

- 	Rerun the pipeline with:
1. 	Changes to the H2O and O2 power?
2. 	And/or adjustments to the telluric_threshold?
- 	Or use the standard star region to correct it from the observed flux?
- Or there is something I am missing?

**Comment by clidman** [comment]
Hi Rami, 
  I suggest that you address this question to 2.3-user-support@mso.anu.edu.au.
